src/sql/parts/objectExplorer/viewlet/objectExplorerActions.ts

```diff
@@ -23,41 +26,51 @@ import { IScriptingService } from 'sql/services/scripting/scriptingService';
 import { IQueryEditorService } from 'sql/parts/query/common/queryEditorService';
 import { IObjectExplorerService } from 'sql/parts/objectExplorer/common/objectExplorerService';
 import * as Constants from 'sql/parts/connection/common/constants';
-import { IWorkbenchEditorService } from 'vs/workbench/services/editor/common/editorService';
-import { ExecuteCommandAction } from 'vs/platform/actions/common/actions';
-import { ICommandService } from 'vs/platform/commands/common/commands';
 import { IConnectionProfile } from 'sql/parts/connection/common/interfaces';
+import { ICapabilitiesService } from 'sql/services/capabilities/capabilitiesService';
+import { ConnectionProfile } from 'sql/parts/connection/common/connectionProfile';
+
 
 export class ObjectExplorerActionsContext {
```

Comment:
shouldn't this inherit from sqlops. ObjectExplorerContext to make sure the syntax matches?

Reply:
Good point. Will make it so